client/profile-wizard/steps/theme/index.js

```diff
@@ -229,6 +229,7 @@ class Theme extends Component {
 								isPrimary
 								onClick={ () => this.onChoose( theme, 'card' ) }
 								isBusy={ chosen === slug }
+								disabled={ chosen === slug }
```

Comment:
We should add this to the `Choose` button as well on line 242. As I got quite a few `theme enabled` notifications when choosing a non-active theme (multi clicking choose).

Reply:
@louwie17 nice catch! 👍  change is in https://github.com/woocommerce/woocommerce-admin/pull/5704/commits/89c7a351006c02e22459f9e85381f1f21fe43ced